# Poses for the three robots of a team
geometry_msgs/Pose2D[3] robot_pose
